packages/@azure/servicebus/data-plane/test/streamingReceiverSessions.spec.ts

```diff
@@ -110,7 +110,7 @@ describe("Streaming Receiver Misc Tests(with sessions)", function(): void {
     await testPeekMsgsLength(messageSession, testMessagesWithSessions.length);
 
     const receivedMsgs: ServiceBusMessage[] = [];
-    messageSession.receive((msgSession: MessageSession, msg: ServiceBusMessage) => {
+    messageSession.receive((msgSession: SessionClient, msg: ServiceBusMessage) => {
```

Comment:
this is totally throwing me off :s 
May need to come up with better names (maybe variable names in tests need to be updated?)

Reply:
`messageSession.receive((msgSession: SessionClient, msg: ServiceBusMessage)`
Can be updated to `receiveClient.receive((x: SessionClient, msg: ServiceBusMessage)`
by making appropriate changes correspondingly.

I still don't know why we pass `msgSession`.